Automattic/jetpack
# VideoPress: restore v5/v6 transform for all platforms

This PR restores VideoPress video block transform from/to core/video v5, for all platforms (Jetpack, Atomic, and Simple)

It was disabled because of the auto-convert process, but considering that the auto-converting process is disabled now, we'd like to provide the users the ability to transform again.

Fixes #

## Proposed changes:

* VideoPress: restore v5/v6 transform for all platforms

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create a VideoPress video instance
* Confirm it's possible to transform from/to v6/v6, in all platforms (Jetpack, Atomic and simple sites)

<img width="893" alt="image" src="https://user-images.githubusercontent.com/77539/219144383-a8e857a0-54ab-4fc4-b063-f89d5992ac43.png">
